schema/dfns.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
I initially thought (looking at the filename) that this was about definitions in specs (because `dfn` is the name of the HTML element), not shared definitions of for JSON schemata. Maybe rename it to definitions?